React2/week2/app/app/counter/CounterContext.js

```diff
@@ -0,0 +1,35 @@
+"use client"
+
+import React, {useReducer, useContext, createContext} from 'react';
+
+const initialState = {count: 0};
+
+function CounterReducer(state, action) {
```

Comment:
It is not a Component, so I would recommend to use lowerCase for naming this function